source/templates.tex

```diff
@@ -1545,6 +1545,48 @@
 the second operand is not checked for satisfaction.
 \end{example}
 
+\pnum
+\begin{note}
+A logical negation expression\iref{expr.unary.op} is an atomic constraint;
+the negation operator is not treated as a logical operation on constraints.
+As a result, distinct negation \grammarterm{constraint-expression}{s}
+that are equivalent under~\ref{temp.over.link}
+do not subsume one another under~\ref{temp.constr.order}.
+Furthermore, if substitution to determine
+whether an atomic constraint is satisfied\iref{temp.constr.atomic}
+encounters a substitution failure, the constraint is not satisfied,
+regardless of the presence of a negation operator.
+It is unclear in principle what the desired value is
+of a substitution failure in a negated concept-id;
```

Comment:
I think it's fine to leave the use of concept-id here since this PR will be merged to master along with CWG Motion 10 (which contains CA096), so no paper will ever be published without the definition for concept-id.   Please unresolve this if you disagree.